### ISSUE TITLE
3.3.1 Error Identification

### ISSUE THREAD

**Issue by WBroadwell** [LabConnect-RCOS/LabConnect-Frontend#76]
- Required inputs or inputs that require a specific format, value, or length provide this information within the element's label.

- [Form validation](https://webaim.org/techniques/formvalidation/) errors are efficient, intuitive, and accessible. The error is clearly identified, quick access to the problematic element is provided, and the user can easily fix the error and resubmit the form.

**Comment by WBroadwell** [comment]
need to recheck after profile and createpost pages are accessible, but added requirments into labels for all input fields I could find. form validation errors should be good for the input fields I found.
